docs/project/requirements/architecture.md

```diff
@@ -0,0 +1,147 @@
+# Architecture
+
+Un schéma d'implémentation détaillé sera établi pour décrire l'architecture globale de l'infrastructure, y compris les différents composants, leur interaction et leur déploiement.
+
+
+## Application
+
+
+### Proxy: Traefik
+
+* Application router
+* High-availability by fault tolerance
+* Load-balancing by requests distribution
+  ALB ?
+* Helm chart
+* Version
+
+
+### Application: FastAPI
+
+* Python
+  * Version:
+  * Packaging
+* Project management? Poetry?
+
+
+### Databases: PostgreSQL
+
+* Helm chart
+* Version
+* Master/slave for high availability and read load balancing
+* Master failover
+
+
+## Infrastructure
+
+
+### Network
+
+
+#### Application Load Balancer (ALB)
```

Comment:
We're using ELB indirectly. It's the EKS that connects our ingress endpoint to ELB.